service/src/test/java/bio/terra/workspace/common/fixtures/ControlledResourceFixtures.java

```diff
@@ -277,7 +268,7 @@ public static ApiGcpAiNotebookInstanceCreationParameters defaultNotebookCreation
     return new ApiGcpAiNotebookInstanceCreationParameters()
         .instanceId("default-instance-id")
         .location("us-east1-b")
-        .machineType("e2-standard-2")
+        .machineType("n1-standard-2")
```

Comment:
From the https://cloud.google.com/compute/docs/general-purpose-machines we know the e2 type does not support Gpu.